client/signup/config/steps-pure.js

```diff
@@ -209,11 +209,9 @@ export function generateSteps( {
 			providesDependencies: [ 'cartItem' ],
 			props: {
 				headerText: i18n.translate( 'Getting ready to launch your website' ),
-				subHeaderText: i18n.translate(
-					"Pick a plan that's right for you. Upgrade it as you grow."
-				),
+				subHeaderText: i18n.translate( "Pick a plan that's right for you. Upgrade as you grow." ),
 				fallbackHeaderText: i18n.translate(
-					"Almost there, pick a plan that's right for you. Upgrade it as you grow."
+					"Almost there, pick a plan that's right for you. Upgrade as you grow."
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.